backend/src/controllers/productControllers.js

```diff
@@ -18,7 +24,7 @@ const addProductToShop = async (req, res) => {
         shop.products.push(newProduct);
         await shop.save();
 
-        logger.info("Product added successfully");
+        logger.info("Product added successfully", newProduct);
```

Comment:
Would appreciate thoughts on what we should and should not log in the DB table. 🙃